credsweeper/rules/config.yaml

```diff
@@ -304,16 +304,15 @@
 
 - name: JSON Web Token
   severity: medium
-  confidence: moderate
+  confidence: strong
```

Comment:
I think it a controversial statement that the JSON encoded in base64 is a security threat